docs/docs/outputs/output-formats.md

```diff
@@ -34,61 +41,40 @@ Tracee supports different output formats for detected events:
+```
 
-Tracee gives user the option to select which files they want to use as standard
-output and standard error:
+!!! Tip
+    A good tip is to pipe **tracee** json output to [jq]() tool, this way
+    you can select fields, rename them, filter values, and many other things:
+    > ```text
+    > sudo ./dist/tracee -o json -o option:parse-arguments
+    > -trace comm=ping | jq -c '. | {eventId, hostName, processName,
+    > hostProcessId,UserId}'
+    > ```
 
-1. Output file
+### GOB
 
-    !!! tip
-        User might use different output formats combined with output file option
+```text
+$ sudo ./dist/tracee --output json --filter comm=bash --filter follow --filter event=openat
+```
 
-    ```text
-    $ sudo ./dist/tracee --filter comm=bash --filter follow --filter event=openat --output json:/tmp/tracee.log
-    
-    $ cat /tmp/tracee.log | jq -c
-    ```
+### GOTEMPLATE
 
-    ```json
-    {"timestamp":1657291487418386000,"threadStartTime":616568205378363,"processorId":11,"processId":1893369,"cgroupId":1,"threadId":1893369,"parentProcessId":3795408,"hostProcessId":1893369,"hostThreadId":1893369,"hostParentProcessId":3795408,"userId":1000,"mountNamespace":4026531840,"pidNamespace":4026531836,"processName":"exa","hostName":"fujitsu","containerId":"","containerImage":"","containerName":"","podName":"","podNamespace":"","podUID":"","eventId":"257","eventName":"openat","argsNum":4,"returnValue":3,"stackAddresses":null,"syscall":"openat","contextFlags":{"containerStarted":false,"isCompat":false},"args":[{"name":"dirfd","type":"int","value":-100},{"name":"pathname","type":"const char*","value":"/etc/ld.so.cache"},{"name":"flags","type":"int","value":524288},{"name":"mode","type":"mode_t","value":0}]}
-    {"timestamp":1657291487418510000,"threadStartTime":616568205378363,"processorId":11,"processId":1893369,"cgroupId":1,"threadId":1893369,"parentProcessId":3795408,"hostProcessId":1893369,"hostThreadId":1893369,"hostParentProcessId":3795408,"userId":1000,"mountNamespace":4026531840,"pidNamespace":4026531836,"processName":"exa","hostName":"fujitsu","containerId":"","containerImage":"","containerName":"","podName":"","podNamespace":"","podUID":"","eventId":"257","eventName":"openat","argsNum":4,"returnValue":3,"stackAddresses":null,"syscall":"openat","contextFlags":{"containerStarted":false,"isCompat":false},"args":[{"name":"dirfd","type":"int","value":-100},{"name":"pathname","type":"const char*","value":"/lib/x86_64-linux-gnu/libgcc_s.so.1"},{"name":"flags","type":"int","value":524288},{"name":"mode","type":"mode_t","value":0}]}
-    ```
+Check [integrations page](../integrating/go-templates.md) for more info.
```

Comment:
this points to an old tarcee-rules docs, can we remove?

Reply:
hmm.. but this page exists in the current docs. should I remove it there?

Reply:
yes, let's remove it. Template output should only exist in classic docs, new binary still not supporting it.

Reply:
I'll update it as well

Reply:
I mean, it is `gotemplate=` and now it belongs on `tracee`, the flags documented here should be enough, maybe add an example here on this page, instead of an extra page for it?